### PR TITLE
TACKLE-801: Assign Credentials to Applications for MTA 6.0

### PR DESCRIPTION
**Pull request by RichardHoch**
MTA 6.0 

Resolves: https://issues.redhat.com/browse/TACKLE-801

Preview:https://deploy-preview-559--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#adding-applications

### CODE REVIEW

File: docs/topics/mta-web-creating-applications.adoc
Comment (istein1) [inline docs/topics/mta-web-creating-applications.adoc:11]: An idea: If this procedure is recommended to be done before creating the profile, then maybe place it in the doc BEFORE the profile creation procedure.

File: docs/topics/mta-web-downloading-app-list-template.adoc
Comment (istein1) [inline docs/topics/mta-web-downloading-app-list-template.adoc:9]: importing

File: docs/topics/mta-web-importing-an-app-list.adoc
Comment (istein1) [inline docs/topics/mta-web-importing-an-app-list.adoc:9]: important->import

File: docs/topics/mta-web-importing-an-app-list.adoc
Comment (istein1) [inline docs/topics/mta-web-importing-an-app-list.adoc:24]: click import was mentioned already in the previous line.

File: docs/topics/mta-web-assigning-application-credentials.adoc
Comment (kpunwatk) [inline docs/topics/mta-web-assigning-application-credentials.adoc:18]: In Analysis Tab there is no Review button we can mention here "**Analyze**" instead 
Screenshot from 2022-11-03 12-30-43

File: docs/topics/mta-web-assigning-application-credentials.adoc
Comment (kpunwatk) [inline docs/topics/mta-web-assigning-application-credentials.adoc:23]: User can select one of each Source Credentials and/or Maven settings from the list

File: docs/topics/mta-web-creating-applications.adoc
Comment (kpunwatk) [inline docs/topics/mta-web-creating-applications.adoc:27]: optional

File: docs/topics/mta-web-creating-applications.adoc
Comment (kpunwatk) [inline docs/topics/mta-web-creating-applications.adoc:31]: The arrow is left of Source Code 
Screenshot from 2022-11-03 13-14-36

File: docs/topics/mta-web-creating-applications.adoc
Comment (kpunwatk) [inline docs/topics/mta-web-creating-applications.adoc:43]: The arrow is left of Binary

File: docs/topics/mta-web-importing-an-app-list.adoc
Comment (kpunwatk) [inline docs/topics/mta-web-importing-an-app-list.adoc:25]: Enqable-->Enable

File: docs/topics/mta-web-about-adding-applications.adoc
Comment (kpunwatk) [inline docs/topics/mta-web-about-adding-applications.adoc:20]: Manually user can fill the Source code and Binary fields

File: docs/topics/mta-web-creating-applications.adoc
Comment (RichardHoch) [reply docs/topics/mta-web-creating-applications.adoc:11]: That's a good idea -- it will have to wait until all the PRs for this guide are merged so i can create a cross-reference.

File: docs/topics/mta-web-importing-an-app-list.adoc
Comment (RichardHoch) [reply docs/topics/mta-web-importing-an-app-list.adoc]: Should probably be Open.

Comment (RichardHoch) [reply docs/topics/mta-web-importing-an-app-list.adoc]: @istein1 Please verify that the button you click when you browse to the file is named  "Open"

File: docs/topics/mta-web-creating-applications.adoc
Comment (istein1) [reply docs/topics/mta-web-creating-applications.adoc:11]: @RichardHoch , ack, thanks.

File: docs/topics/mta-web-creating-applications.adoc
Comment (sshveta) [inline docs/topics/mta-web-creating-applications.adoc:9]: spelling : Inventory

**Review by sshveta — CHANGES_REQUESTED** [review]

File: docs/topics/mta-web-creating-applications.adoc
Comment (RichardHoch) [reply docs/topics/mta-web-creating-applications.adoc]: fixed

Comment (RichardHoch) [reply docs/topics/mta-web-creating-applications.adoc]: @sshveta  I fixed this -- please review the PR again.

**Comment by RichardHoch** [comment]
@sbeskin-redhat please review.

**Review by aufi — COMMENTED** [review]
LGTM!

**Pull request merged**
